src/platform/shared.c

```diff
@@ -58,6 +59,7 @@ SDL_Color gPalette[256];
 uint32 gPaletteHWMapped[256];
 
 bool gSteamOverlayActive = false;
+Uint32 gOpenRCT2EventType;
```

Comment:
Is this just representing `openrct2://` events? If so then it should be called `gOpenRCT2UriEventType` or something.